### PR TITLE
[MB-1314] NullPointerException when subscriber closes connection without consuming messages

### PR DESCRIPTION
**Pull request by a5anka**
Initialize the message flusher when slot delivery worker is given a
destination to deliver messages.

https://wso2.org/jira/browse/MB-1314

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessageFlusher.java [hunk 1]
```diff
@@ -233,6 +233,22 @@ public MessageDeliveryInfo getMessageDeliveryInfo(String destination) throws And
         return messageDeliveryInfo;
     }
 
+    /**
+     * Initialize message flusher for delivering messages for the given destination
+     *
+     * @param destination
```
Comment (ramith) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessageFlusher.java]: is this a storage queue name?

Comment (a5anka) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessageFlusher.java]: No. It is the message destination.

Comment (ramith) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessageFlusher.java]: documentation?

Comment (a5anka) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessageFlusher.java]: Added

**Pull request merged**